src/sentry/constants.py

```diff
@@ -263,13 +263,32 @@ def _load_platform_data():
 
 # special cases where the marketing slug differs from the integration id
 # (in _platforms.json). missing values (for example: "java") should assume
-# the marketing slug is the same as the integration id.
+# the marketing slug is the same as the integration id:
+# javascript, node, python, php, ruby, go, swift, objc, java, perl, elixir
 MARKETING_SLUG_TO_INTEGRATION_ID = {
-    "kotlin": "java",
-    "scala": "java",
+    "kotlin": "java-android",
+    "scala": "java-logback",
```

Comment:
Thanks, done!

Would be nice to eventually have docs for Kotlin, even if they're basically, "it just works with Java stuff."